js/hyperswitch-hyperservice.js

```diff
@@ -290,50 +290,54 @@ document.addEventListener("DOMContentLoaded", () => {
 });
 
 async function hyperswitchPaymentHandleSubmit(isOneClickPaymentMethod, result) {
-  if (result || isOneClickPaymentMethod) {
-    let err;
-    if (!isOneClickPaymentMethod && result) {
-      const { error } = await hyper.confirmPayment({
-        confirmParams: {
-          return_url: hyperswitchReturnUrl,
-        },
+  if (!result && !isOneClickPaymentMethod) return;
```

Comment:
@nabeel001 Please change back the if conditions as using positive logic (like a || b) tends to be clearer because it tells the reader exactly when the action will happen, without needing to mentally reverse a negation.

Reply:
@sakksham7 
I have made the requested changes. Kindly verify.